CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-For information about how to contribute to the Void documentation, please read the following sections of the Void Handbook:
+To contribute to the Void documentation, please read the following sections of the Void Handbook. Pull requests that do not meet the criteria described in these sections will not be merged.
```

Comment:
Agree with the sentiment, but would like to mention that we are open to helping make PRs fit it. Or do you think that's clear already?

Reply:
To manage our own workload, i'd like to encourage users to meet as many of the criteria as possible before submitting a PR. So how about something like:

> will not be merged. Please try to address as many of the criteria as possible before submitting a PR - that will improve the chances of us being able to include your contribution quickly.

Reply:
Love it! Would prefer passive voice for the last part: "will improve the chances of your contribution being accepted quickly"

Reply:
Yes, that's better - PR updated.

Reply:
Thanks!